### PR TITLE
[ETL-275] Add recover development account under ScienceDevOU

### PR DESCRIPTION
**Pull request by rxu17**
Hello,
I'm opening this PR to create a development AWS account. This development account paired with the production account will house all the ETL work required for the RECOVER project, such that all the work done in these accounts will be charged to the RECOVER grant. 

See [PR #650](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/650) for the PR creating the corresponding production account.

PR Checklist:
[X] Describe and explain your intentions for this change
[X] Setup pre-commit and run the validators (info in README.md)

### CODE REVIEW

**Comment by thomasyu888** [comment]
So I think Brad had suggested we also include the dev account under production accounts. Or maybe there needs to be a new category of FISMA Accounts

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -563,6 +564,22 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  RecoverDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-recover-dev
+      AccountId: '634761300905'
+      RootEmail: aws-recover-dev@sagebase.org
```
[the remark this answers is not in the document]
Comment (thomasyu888) [reply org-formation/organization.yaml:572]: Done

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -563,6 +564,22 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  RecoverDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-recover-dev
+      AccountId: '634761300905'
```
[the remark this answers is not in the document]
Comment (rxu17) [reply org-formation/organization.yaml]: Done

File: org-formation/organization.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (brucehoff) [reply org-formation/organization.yaml:571]: Please note that "who can post" needs to be set to "anyone on the web" lest AWS is unable to send emails to the alias: https://groups.google.com/a/sagebase.org/g/aws-recover-dev/settings?hl=en

**Comment by brucehoff** [comment]
LGTM but I'd like @zaro0508 to approve since this is my first time reviewing an account creation PR.

File: org-formation/organization.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomasyu888) [reply org-formation/organization.yaml:571]: Done! Thanks!

**Review by brucehoff — APPROVED** [review]

**Comment by brucehoff** [comment]
@rxu17 The PR says, "This branch has conflicts that must be resolved".  Would you please have a look?

**Comment by rxu17** [comment]
Conflicts resolved

**Pull request merged**